templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,42 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List
+  </h1>
+  <p><h5><a class="link" href="{% url 'taxi:manufacturer-create' %}">Click here to add a new manufacturer</a></h5></p>
 
-    {% if manufacturer_list %}
-      <table class="table">
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+
+      {% for manufacturer in manufacturer_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
+          <td>
+            {{ manufacturer.id }}
+          </td>
+          <td>
+            {{ manufacturer.name }}
+          </td>
+          <td>
+            {{ manufacturer.country }}
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-update' manufacturer.id %}">UPDATE</a>
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">DELETE</a>
```

Comment:
Code Style: According to the checklist, you should add a `Cancel` button that leads to the previous page the user was on. This provides a better user experience.